docs/solutions/hypervisors/kvm/kvm-baremetal/datasource.tf

```diff
@@ -0,0 +1,27 @@
+data "oci_identity_availability_domains" "ads" {
+  compartment_id = "${var.tenancy_ocid}"
+}
+
+data "oci_core_images" "base-image" {
+  compartment_id = "${var.compartment_ocid}"
+  display_name   = "Oracle-Linux-7.4-2017.10.25-0"
+}
+
+data "oci_core_shape" "supported_shapes" {
```

Comment:
Is the goal here to ensure that users are using BM shapes? This doesn't seem necessary to me - I would just have them specify the shape directly and document that it must be BM (as is already done).

Reply:
Yes, this is a safe-guard - just  to ensure users are using the appropriate shape, otherwise launching the KVM domain fails without any hint. That happened to me when I was modifying the code and set the wrong shape value for the nested kvm.